libraries/botbuilder-applicationinsights/tests/telemetryInitializer.test.js

```diff
@@ -4,6 +4,9 @@
 const assert = require('assert');
```

Comment:
New tests are at the end (line 123). Everything else is no-op, mostly changing `var` to `const` because `var` hoisting isn't intuitive.